homework.rb

```diff
@@ -1,6 +1,10 @@
-VARIANT = 0
+VARIANT = 8
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  letters = *("A".."Z")
+
+  letter1, letter2 = 2.times.map { letters.sample(3).join }
```

Comment:
no variables just long oneliner is ok? 
just tested the speed is better.
`[letters.sample(3).join, rand(100..999).to_s, letters.sample(3).join, rand(100..999).to_s].join("-")`